### ISSUE TITLE
Active page state not announced by screen reader nav links

### ISSUE THREAD

**Issue by SatyamSetia** [AccessibleForAll/AccessibleWebDev#343]
**Describe the bug**
Nav links in the page does not have attribute which can define their state (is it current page or not) for screen reader.

**To Reproduce**
1. Go to Home page and use screen reader
2. Use keyboard to shift focus to other nav links and try coming back to active link.

**Expected behavior**
Screen reader should announce if focus is present on active page nav link.

**Screenshots**
Current experience on screen reader - 

https://github.com/AccessibleForAll/AccessibleWebDev/assets/17885747/0e0fcfa1-a62b-4c35-a61c-29b21893192b

Expected experience on screen reader - 

https://github.com/AccessibleForAll/AccessibleWebDev/assets/17885747/5378918f-b59a-43d7-9438-ec0e2814cf56

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Comment by SatyamSetia** [comment]
Hey, @EmmaDawsonDev
Just wanted to share my interest on working this bug fix if in-case you find it to be a relevant issue to be fixed.

My proposal for the fix -
Adding aria-current to the anchor tags (nav links) with value - "page" (if it is current) OR false

**Comment by EmmaDawsonDev** [comment]
You are correct, I also noticed this was missing recently so thanks for raising the issue. I will assign it to you 😊

**Comment by SatyamSetia** [comment]
Acknowledged 👍
